### PR TITLE
Add dictionary to run with fuzzers

### PR DESCRIPTION
**Pull request by CarlosEduR**
Relates to: https://github.com/ada-url/ada/pull/658

### CODE REVIEW

File: fuzz/url.dict [hunk 1]
```diff
@@ -0,0 +1,2 @@
+"https://"
```
Comment (anonrig) [inline fuzz/url.dict:1]: I think you should add all types of protocols (available as special schemes in url spec)

**Review by anonrig — APPROVED** [review]

**Comment by anonrig** [comment]
Is there anything that needs to be done to pass the dict to the fuzzer?

File: fuzz/url.dict (hunk 1 above)
Comment (CarlosEduR) [reply fuzz/url.dict:1]: Ok, I'll do it

**Comment by CarlosEduR** [comment]
> Is there anything that needs to be done to pass the dict to the fuzzer?

From what I was reading here, looks we just need this .options file.
image

I saw some other examples, just updating the build.sh to copy these files too and we should be fine. But let's see... 
https://github.com/google/oss-fuzz/blob/ad88a2e5295d91251d15f8a612758cd9e5ad92db/projects/expat/build.sh

File: fuzz/ada_fuzzer.options [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
+dict = url.dict
```
Comment (anonrig) [inline fuzz/ada_fuzzer.options:2]: Can you add ending missing newlines to each of these files? editorconfog extension for your ide should fix this

Comment (CarlosEduR) [reply fuzz/ada_fuzzer.options:2]: hmm, will update it. what ide do you use?

Comment (anonrig) [reply fuzz/ada_fuzzer.options:2]: zed and clion

**Comment by CarlosEduR** [comment]
@anonrig would you like to set a max value here?
image

From examples, expat defined one but other projects were not.

**Comment by anonrig** [comment]
We can do 1024

**Comment by CarlosEduR** [comment]
@anonrig have you noticed any difference since last run? I haven't

**Comment by anonrig** [comment]
> @anonrig have you noticed any difference since last run? I haven't

I haven't as well

**Pull request merged**